api_proxy.yaml

```diff
@@ -426,9 +426,12 @@ definitions:
         type: string
       spidLevel:
          $ref: '#/definitions/SpidLevel'
+      walletToken:
```

Comment:
@cloudify just noticed these specs use camelCase while the functions one are snake_case,
is it a big deal ?

Reply:
not a big deal imho since these APIs are private